Added support for showing latest reported cases

jeremyphilemon opened the pull request:
🤷‍♂ 

<img width="1483" alt="Screen Shot 2020-04-03 at 5 17 05 PM" src="https://user-images.githubusercontent.com/17938322/78370204-045b6500-75cf-11ea-887e-74159db526ff.png">

jeremyphilemon committed:
Added latest cases on frontpage.

vramana commented on src/components/patients.js:
```diff
@@ -53,35 +54,40 @@ function Patients(props) {
+          .map((day, index) => {
+            if (day !== 'Invalid Date') {
+              return (
+                <React.Fragment>
```
`key` prop is missing

jeremyphilemon replied:
Haha fixed :)

jeremyphilemon committed:
Fixed alignments.

jeremyphilemon committed:
Color coded by gender.

jeremyphilemon committed:
Redesigned the layout.

shuklaayush committed:
Fixes to patient database [WIP] (#570)

* Clean up code; add hierarcy to filters
* Use copy of data in filteredPatients
* Fix bug where single patient was not displayed
* More fixes to selection boxes
  - Hide boxes which are not relevant
  - Default to first selection
  - Comment out date selection for now

jeremyphilemon committed:
Patients db ready for release yo.
